### DISCUSSION TITLE
Who plays the lead role in the 1976 movie Rocky?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#631] [Galaxy Trivia] [answered]
**Bonus:** How many official sequels are there?

**Comment by peterbaccaro** [comment, marked as answer]
The lead role in the 1976 movie "Rocky" is played by **Sylvester Stallone**.

There are a total of nine official Rocky sequels. 

1) Rocky II (1979)
2) Rocky III (1982)
3) Rocky IV (1985)
4) Rocky V (1990)
5) Rocky Balboa (2006)
6) Creed (2015)
7) Creed II (2018)
8) Rocky IV: Rocky vs. Drago - The Ultimate Director's Cut (2021)
9) Creed III (2023)

**Reply by ParanoidUser** [reply]
Interesting, never heard of Rocky vs. Drago 🎬. Thanks for pointing that out!
